.github/workflows/main.yml

```
@@ -171,7 +171,9 @@ jobs:
       # NOTE: edited to run checks for python package
       - name: Run tests
         run: |
+          python -m unittest tests.test_main_system_caches
           pytest -m 'vetiver'
```

```suggestion
          pytest tests/test_main_system_caches.py
          pytest -m 'vetiver'
```

This should also work.